controllers/metal3.io/host_state_machine_test.go

```diff
@@ -1403,7 +1403,7 @@ func (p *mockProvisioner) GetFirmwareComponents() (components []metal3api.Firmwa
 	return components, nil
 }
 
-func (m *mockProvisioner) IsDataImageReady() (isNodeBusy bool, nodeError error) {
+func (p *mockProvisioner) GetDataImageStatus() (isImageAttached bool, err error) {
```

Comment:
I'm wondering if renaming is ok here, since it was called IsDataImageReady before..

Reply:
I think the two functions served similar but distinct roles. "IsDataImageReady" was meant to capture node errors and guess if the virtual media attach/detach was successful, while "GetDataImageStatus" gets the actual current status for virtual media.

Reply:
Provisioner API is not public, at least for now. Renaming is fine.